### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bythewaters**

### CODE REVIEW

File: backend/cinema_service/settings.py [hunk 1]
```diff
@@ -175,3 +178,7 @@
     "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
     "ROTATE_REFRESH_TOKENS": False,
 }
+
+CORS_ALLOWED_ORIGINS = [
+    "http://127.0.0.1:5173",
+]
```
Comment (EdAlekseiev) [inline backend/cinema_service/settings.py:184]: I advise moving this value to .env file too

File: docker-compose.yml [hunk 2]
```diff
@@ -20,8 +20,9 @@ services:
       - db
 
   db:
+    hostname: db
     image: postgres:14-alpine
     ports:
-      - "5432:5432"
+      - "5433:5432"
     env_file:
       - .env
```
Comment (EdAlekseiev) [inline docker-compose.yml:28]: Add new line

File: docker-compose.yml [hunk 3]
```diff
@@ -20,8 +20,9 @@ services:
       - db
 
   db:
+    hostname: db
     image: postgres:14-alpine
     ports:
-      - "5432:5432"
```
Comment (EdAlekseiev) [inline docker-compose.yml:25]: Why do you change the port? 5432 is the standard port for postgres

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (bythewaters) [reply docker-compose.yml:25]: I have PostgreSQL installed on my computer, and I can't connect to db, cos port 5432 default in local db.

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**